packages/javascript/src/index.ts

```diff
@@ -32,7 +32,7 @@ export default class Appsignal {
   }
 
   private _env = Environment.serialize()
-  private _queue = new Queue((window as any).__APPSIGNAL_QUEUE__ || [])
```

Comment:
`window.__APPSIGNAL_QUEUE__` is a relic from when we were going to cache errors that were thrown before load, it was never used.